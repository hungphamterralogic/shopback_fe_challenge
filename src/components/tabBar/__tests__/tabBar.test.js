import React from 'react';
import { genInitialMockStore, renderWithStore } from 'test/reduxComponentTestHelpers';

import TabBar from '../tabBar';

const categories = [
  { id: 'id01', name: 'name01', iconUrl: 'iconUrl01' },
  { id: 'id02', name: 'name02', iconUrl: 'iconUrl02' },
  { id: 'id03', name: 'name03', iconUrl: 'iconUrl03' },
  { id: 'id04', name: 'name04', iconUrl: 'iconUrl04' }
];

describe('TabBar', () => {
  test('display properly', () => {
    const mockStore = genInitialMockStore();
    const renderer = renderWithStore(<TabBar categories={categories} />, mockStore);
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
