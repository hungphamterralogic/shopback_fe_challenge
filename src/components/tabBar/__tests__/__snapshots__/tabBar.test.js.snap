// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBar display properly 1`] = `
<div
  className="TabBar-tabBar-0-1-1"
>
  <div
    className="TabItem-tabItem-0-1-4 TabItem-tabItemDestop-0-1-5"
    onClick={[Function]}
  >
    <img
      alt="icon"
      className="TabItem-icon-0-1-8"
      src="iconUrl01"
    />
    <span
      className="TabItem-name-0-1-9"
    >
      name01
    </span>
  </div>
  <div
    className="TabItem-tabItem-0-1-4 TabItem-tabItemDestop-0-1-5"
    onClick={[Function]}
  >
    <img
      alt="icon"
      className="TabItem-icon-0-1-8"
      src="iconUrl02"
    />
    <span
      className="TabItem-name-0-1-9"
    >
      name02
    </span>
  </div>
  <div
    className="TabItem-tabItem-0-1-4 TabItem-tabItemDestop-0-1-5"
    onClick={[Function]}
  >
    <img
      alt="icon"
      className="TabItem-icon-0-1-8"
      src="iconUrl03"
    />
    <span
      className="TabItem-name-0-1-9"
    >
      name03
    </span>
  </div>
  <div
    className="TabItem-tabItem-0-1-4 TabItem-tabItemDestop-0-1-5"
    onClick={[Function]}
  >
    <img
      alt="icon"
      className="TabItem-icon-0-1-8"
      src="iconUrl04"
    />
    <span
      className="TabItem-name-0-1-9"
    >
      name04
    </span>
  </div>
</div>
`;
